LEAF_Request_Portal/sources/Group.php

```diff
@@ -269,20 +269,23 @@ public function getMembers($groupID, bool $searchDeleted = false): array|string
      *
      * @return string
      */
-    public function addMember($member, $groupID): string
+    public function addMember(string $member, int $groupID): array
```

Comment:
Is this a function we can easily update to the preferred "json" return?

Reply:
This is returning the properly formatted json response but php doesn't have json as a type hint. The only thing that could be done differently here would be to json_encode and send it back as a string. But I think it would be better to send the array here and if it's actually an endpoint have the endpoint json_encode it before sending it back to the caller.